LaCE-onboarding-guide.md

```diff
@@ -37,31 +37,60 @@ This page will help you solve common queries that you may face while working on
     - Difficulty
     - Problem
     - Interaction
-    - Answers ans Responses
+    - Answers and Responses
     - Hints
   - Once all these sections are filled, the question can be saved.
 
 ## 3. Generating data on localhost
-- To generate data, we must first navigate to the admin page. We will automatically be redirected to the `ACTIVITIES` tab.
+- To generate data, we must first navigate to the admin page.
+
+![Admin Page](https://github.com/prafulbbandre/oppia-web-developer-docs/assets/109658545/bbfb7acd-356f-41a1-8f4e-f7747a9de805)
+
+-  We will automatically be redirected to the `ACTIVITIES` tab.
 - This tab will have several sections:
   - Reload a single exploration: This section has preloaded explorations. These can be loaded onto your local server by clicking the `Reload` button for each of them.
+
+![Activities Tab](https://github.com/prafulbbandre/oppia-web-developer-docs/assets/109658545/39a77f42-c38b-4f5b-9bd9-29366ab8bcb8)
```

Comment:
All the images seems to be linked to your fork. Can you move them to the oppia/oppia-web-developer-docs repo?